features/routing/idle.feature

```diff
@@ -4,28 +4,28 @@ Feature: idle service related scenarios
   # @case_id OCP-10935
   Scenario: Pod can be changed to un-idle when there is unsecure or edge or passthrough route coming
     Given I have a project
-    Given I obtain test data file "routing/list_for_caddy.json"
-    When I run oc create over "list_for_caddy.json" replacing paths:
-      | ["items"][0]["spec"]["replicas"] | 1 |
+    Given I obtain test data file "routing/web-server-rc.yaml"
+    When I run the :create client command with:
+      | f | web-server-rc.yaml |
```

Comment:
the new web-server-rc.yaml uses `replicas: 1` as default, replacing paths is not required.